### ISSUE TITLE
cannot start mjolnir v1.3.0

### ISSUE THREAD

**Issue by cremesk** [matrix-org/mjolnir#206]
**Describe the bug**
```
Feb 01 11:27:10 matrix.envs.net systemd[1]: Started matrix-mjolnir.
Feb 01 11:27:10 matrix.envs.net node[330050]: Creating Web APIs
Feb 01 11:27:10 matrix.envs.net node[330050]: Starting web server
Feb 01 11:27:10 matrix.envs.net node[330050]: Configuring /api/1/report/:room_id/:event_id...
Feb 01 11:27:10 matrix.envs.net node[330050]: Configuring /api/1/report/:room_id/:event_id... DONE
Feb 01 11:27:10 matrix.envs.net node[330050]: Tue, 01 Feb 2022 11:27:10 GMT [ERROR] [Mjolnir] Error during startup:
Feb 01 11:27:10 matrix.envs.net node[330050]: Tue, 01 Feb 2022 11:27:10 GMT [ERROR] [Mjolnir] TypeError: Cannot read property 'enabled' of undefined
Feb 01 11:27:10 matrix.envs.net node[330050]:     at WebAPIs.<anonymous> (/opt/mjolnir/lib/webapis/WebAPIs.js:65:49)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at Generator.next (<anonymous>)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at /opt/mjolnir/lib/webapis/WebAPIs.js:23:71
Feb 01 11:27:10 matrix.envs.net node[330050]:     at new Promise (<anonymous>)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at __awaiter (/opt/mjolnir/lib/webapis/WebAPIs.js:19:12)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at WebAPIs.start (/opt/mjolnir/lib/webapis/WebAPIs.js:48:16)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at Mjolnir.<anonymous> (/opt/mjolnir/lib/Mjolnir.js:243:36)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at Generator.next (<anonymous>)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at fulfilled (/opt/mjolnir/lib/Mjolnir.js:20:58)
Feb 01 11:27:10 matrix.envs.net node[330050]:     at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

**Info:**
 - OS: Debian 11
 - nodejs v14.18.3

**Comment by Gnuxie** [comment]
oops, sorry about that, if you have the abuse reporting enabled you will need to also add

```
  ruleServer:
    enabled: false
```

under `web:` in your config for the meantime

**Comment by cremesk** [comment]
thanks! it fixed the issue and the mjolnir v1.3.0 service started now.
